### PR TITLE
Hotfix: Re-trigger exit queries on wallet change

### PR DESCRIPTION
**Pull request by garethfuller**
# Description

We should re-trigger exit queries, in particular, the single exit max query, when the user's wallet changes.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## How should this be tested?

- [ ] Switch wallets in the single asset exit flow and check if the max amount to withdraw updates.

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have requested at least 1 review (If the PR is significant enough, use best judgement here)
- [x] I have commented my code where relevant, particularly in hard-to-understand areas
- [x] If package-lock.json has changes, it was intentional.
- [x] The base of this PR is `master` if hotfix, `develop` if not

### CODE REVIEW

File: src/providers/local/exit-pool.provider.ts [hunk 1]
```diff
@@ -97,17 +97,14 @@ export const exitPoolProvider = (
...
+  const { account, getSigner } = useWeb3();
   const { relayerSignature, relayerApprovalAction, relayerApprovalTx } =
     useRelayerApproval(RelayerType.BATCH);
 
   const debounceQueryExit = debounce(queryExit, debounceQueryExitMillis);
   const debounceGetSingleAssetMax = debounce(
     getSingleAssetMax,
-    debounceGetSingleAssetMaxMillis,
-    {
-      leading: true,
```
Comment (timjrobinson) [inline src/providers/local/exit-pool.provider.ts:109]: Why was this needed before but isn't needed now?

**Review by timjrobinson — APPROVED** [review]

File: src/providers/local/exit-pool.provider.ts (hunk 1 above)
Comment (garethfuller) [reply src/providers/local/exit-pool.provider.ts:109]: I couldn't figure that out, I removed it to see if it would fix the problem. Let me double-check if this needs to be removed.

Comment (garethfuller) [reply src/providers/local/exit-pool.provider.ts]: Yeah, this wasn't related to the issue, so I've reverted.
